charmcraft/commands/store/__init__.py

```diff
@@ -420,14 +433,25 @@ def fill_parser(self, parser):
         parser.add_argument(
             '-c', '--channel', action='append', required=True,
             help="The channel(s) to release to (this option can be indicated multiple times)")
+        parser.add_argument(
+            '--resource', action='append', type=ResourceOption(), default=[],
+            help=(
+                "The resource(s)s to attach to to the release, in the <name>:<revision> format "
```

Comment:
There's an extra "s" here in resource(s)s.

Reply:
Thanks! will fix it in the next PR.